tests/test_orquestra_device.py

```diff
@@ -249,14 +249,19 @@ def circuit():
             qml.Hadamard(wires=[0])
             return qml.expval(qml.Hadamard(0))
 
-        qnode = qml.QNode(circuit, dev)
-        qnode._construct([], {})
 
-        qasm = dev.serialize_circuit(qnode.circuit)
+        with monkeypatch.context() as m:
+            m.setattr(dev, "execute", lambda *args, **kwargs: np.array([None]))
+            qnode = qml.QNode(circuit, dev)
+            qnode()
+
+        circuit = qnode.qtape.graph if qml.tape_mode_active() else qnode.circuit
```

Comment:
I suggest not having `qml.tape_mode_active()` here as it is temporary and will be removed from PL
```suggestion
        circuit = qnode.qtape.graph if hasttr(qnode, "qtape") else qnode.circuit
```